tests/test_vb_ARD.py

```diff
@@ -1,119 +1,86 @@
 import numpy as np
-import unittest
-from bayes.vb import *
-from test_vb import ModelError, ModelErrorWithJacobian
+import pytest
```

Comment:
Is the ARD test case now working (I mean really giving reasonable results, not that the tests return True)?

Reply:
I'm not fully sure. The checks fail, if you remove the `index_ARD` option. #79 continues to investigate it.